To: vim_dev@googlegroups.com
Subject: Patch 7.3.089
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.089
Problem:    Compiler warning on 64 bit MS-Windows.
Solution:   Add type cast. (Mike Williams)
Files:	    src/netbeans.c


*** ../vim-7.3.088/src/netbeans.c	2010-12-17 18:06:00.000000000 +0100
--- src/netbeans.c	2010-12-17 20:21:05.000000000 +0100
***************
*** 800,806 ****
  	 *		    -> gui event loop or select loop
  	 *			-> netbeans_read()
  	 */
! 	save((char_u *)DETACH_MSG, strlen(DETACH_MSG));
  	nb_close_socket();
  
  	if (len < 0)
--- 800,806 ----
  	 *		    -> gui event loop or select loop
  	 *			-> netbeans_read()
  	 */
! 	save((char_u *)DETACH_MSG, (int)strlen(DETACH_MSG));
  	nb_close_socket();
  
  	if (len < 0)
*** ../vim-7.3.088/src/version.c	2010-12-24 13:39:29.000000000 +0100
--- src/version.c	2010-12-24 13:59:03.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     89,
  /**/

-- 
The 50-50-90 rule: Anytime you have a 50-50 chance of getting
something right, there's a 90% probability you'll get it wrong.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
